src/v2/styles/MessageReactions/MessageReactions-layout.scss

```diff
@@ -4,6 +4,8 @@
   display: flex;
 
   .str-chat__message-reactions {
+    overflow-y: hidden;
+    overflow-x: auto;
```

Comment:
Do we need to make some accommodations for Windows, where scroll bars are always visible and occupy additional space?

Reply:
Yep, it looks a bit weird on Windows:
![image](https://github.com/GetStream/stream-chat-css/assets/975978/319e6b0b-dff3-427d-bdce-5d56cf6c52e2)

Reply:
hm, I thought the scrollbar is only visible on windows if the content needs scrolling, but it looks like it's there even if there is no need for the scroll?

Reply:
This is the definition for `auto`: Overflow content is clipped at the element's padding box, and overflow content can be scrolled into view. Unlike scroll, user agents display scroll bars only if the content is overflowing and hide scroll bars by default. If content fits inside the element's padding box, it looks the same as with visible, but still establishes a new block-formatting context. Desktop browsers provide scroll bars if content overflows. Does it work differently on windows?

Reply:
I did a test in windows and this is what I'm seeing:

UI when the reaction list fits the available space - no scrollbar here
![2024-02-01 10 41 46](https://github.com/GetStream/stream-chat-css/assets/6690098/65164a60-cb7d-4798-b016-d04765047f17)

UI when the reaction list doesn't fit the screen - before the fix, overflow
![2024-02-01 10 41 57](https://github.com/GetStream/stream-chat-css/assets/6690098/25af62cc-f1a4-49e2-9a66-ad477ef88774)

UI when the reaction list doesn't fit the screen - after the fix, scrollbar
![2024-02-01 10 41 53](https://github.com/GetStream/stream-chat-css/assets/6690098/52e868a2-bb16-4e9c-b0a6-a38fdd9a5459)

I agree that the scrollbar doesn't look nice, and if you have an easy to implement idea, I'm happy to change my fix. But if we don't have that, I think we should apply that fix, because it's better to have an ugly scrollbar, than an overflow.

Reply:
I agree that some scrollbar is better than no scrollbar :) In my screenshot there are actually additional reactions overflowing the container, that's why the scrollbar is visible.

Here's a usual way deal to deal with visible scrollbars:

```css
/* On the reactions list container, hide all overflow */
.str-chat__message-reactions-container {
  overflow: hidden;
}

.str-chat__message-reactions {
  overflow-x: scroll;
  overflow-y: hidden;

  /* On the reactions list, give it some additional height.
     Normally the height is 28px, let's make it 56px */
  height: 56px;
  box-sizing: border-box;
  align-items: start; /* prevent reactions from stretching to 56px height */

  /* Hide all "additional" hide, including a scrollbar,
     in parent's overflow */
  margin-bottom: -28px;
}
```

This way even when the scrollbar is displayed, it stay hidden in its parent overflow.